lib/generator._js

```diff
@@ -214,12 +214,17 @@ ScriptGenerator.prototype.generateNodeDeploymentScript = function (_) {
 ScriptGenerator.prototype.generateFunctionAppDeploymentScript = function(_) {
   log.info('Generating deployment script for function App');
 
-  if (this.scriptType != ScriptType.batch) {
-    throw new Error('Only batch script files are supported for function App');
-  }
-
   options = {}
   if (this.solutionPath) {
+    // if it has a .sln, it must be targeting windows
+    // as for now function CLI does not create function app targeting netstandard
+    // although runtime support is already there:
+    // https://github.com/Azure/azure-functions-vs-build-sdk/tree/master/sample/FunctionAppNETStandard
+    // https://github.com/Azure/azure-functions-vs-build-sdk/blob/master/build.cmd#L69
+    // TODO add msbuild template for bash
+    if (this.scriptType == ScriptType.bash) {
+      throw new Error('function app targeting dotnet core is currently not supported');
```

Comment:
I thought the main point of doing this was to support dotnet core on Linux. Is there really a demand for using bash script in the Functions 1.x world, which is not cross platform?